src/language-vue/parser-vue.js

```diff
@@ -150,7 +150,11 @@ function parseHTML(html, options) {
       let text;
       let rest;
       let next;
+<<<<<<< pr-fixes
+      
+=======
 
+>>>>>>> master
```

Comment:
It seems you forgot to handle this merge conflict.

Reply:
Oh Yes !! My Bad! :) Fixed